frontend/src/components/editor/RecoveryButton.tsx

```diff
@@ -1,10 +1,10 @@
 /* Copyright 2024 Marimo. All rights reserved. */
 
 import type { Notebook } from "@marimo-team/marimo-api";
-import { Button } from "components/ui/button";
```

Comment:
These kind of imports were technically allowed because `baseUrl` was set to `src`, but the new config requires any "root" imports are imported with `@/*` prefix. Better to be consistent.